tests/integration/python/test_python.py

```diff
@@ -20,16 +20,27 @@ def models(self):
     def project_config(self):
         return {
             "config-version": 2,
-            "vars": {"http_path": os.getenv("DBT_DATABRICKS_CLUSTER_HTTP_PATH")},
+            "vars": {
+                "http_path": os.getenv("DBT_DATABRICKS_CLUSTER_HTTP_PATH"),
+                "location_root": "",
```

Comment:
Yes, because dbt evaluates the whole project for Jinja compilation, so if this is missing it will fail, even though the model we're building doesn't need it.  Note: this is specific to this test, because of the var interpolation, not something that users would hit.  I could try to come up with a better solution for defaulting the location_root variable, but basically all of the vars need to be defined in order to compile.

Reply:
Point is moot.  I'm dropping location_root from the integration test as too costly to maintain.  I've manually validated that it works E2E, and we can get coverage of this line through unit testing the macro.